### PR TITLE
User resource

### PR DESCRIPTION
**Pull request by deanhuynh**
Supports users and invites. Also cleans up labels to no longer have an unnecessary ID attribute.

### CODE REVIEW

File: internal/provider/user_resource_test.go [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+					"data": {
+						"status": "SUCCESS"
+					}
+				}
+				`
+			} else if req.URL.Path == "/users" && req.Method == http.MethodGet {
+				if inviteQueried < 6 {
+					payload = `
+					{
+						"data": {
```
Comment (marcelopv) [inline internal/provider/user_resource_test.go:44]: Missing formatting or just github acting weird with formatting?

File: internal/provider/user_resource_test.go
Comment (marcelopv) [inline internal/provider/user_resource_test.go:1]: Missing formatting or just github acting weird with formatting?

File: internal/provider/user_resource_test.go [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+	t.Parallel()
+
+	updated := 0
+	inviteQueried := 0
+	fakeServer := httptest.NewServer(
+		http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+			w.Header().Set("content-type", "application/json")
+			var payload string
+
+			// Rules requests
```
Comment (marcelopv) [inline internal/provider/user_resource_test.go:21]:
```suggestion
```

File: internal/provider/user_resource_test.go
Comment (deanhuynh) [reply internal/provider/user_resource_test.go:1]: Github is acting weird. It's formatted nicely in the editor.

**Review by marcelopv — APPROVED** [review]

**Pull request merged**